src/dbt/include/deltastream/macros/adapters.sql

```diff
@@ -20,7 +20,12 @@
   as {{ sql }};
 {%- endmacro %}
 
-{% macro deltastream__create_table_as(relation, sql, parameters) -%}
+{# dbt standard create_table_as macro override #}
+{% macro deltastream__create_table_as(temporary, relation, compiled_code) -%}
```

Comment:
`create_table_as` macro was calling with these parameters which doesn't match the ones I used in `deltastream__create_deltastream_table_as` which "collided" with the override strategy.